src/api/alchemy-websocket-provider.ts

```diff
@@ -1008,13 +1003,57 @@ function addToPastEventsBuffer<T>(
   pastEvents.push(event);
 }
 
-function isAlchemyEvent(event: AlchemyEventType): event is object {
+function isAlchemyEvent(
+  event: AlchemyEventType
+): event is AlchemyPendingTransactionsEventFilter {
   return typeof event === 'object' && 'method' in event;
 }
 
-function getAlchemyEventTag(event: AlchemyEventType): string {
+/**
+ * Creates a string representation of an `alchemy_pendingTransaction`
+ * subscription filter that is compatible with the ethers implementation of event tag.
+ *
```

Comment:
Can we link to documentation of Ethers implementation? Or did you just have to reverse engineer this